src/app/home/home.component.html

```diff
@@ -1,4 +1,5 @@
 <sw-banner swScrollSpy id="banner"></sw-banner>
+<sw-countdown swScrollSpy id="countDown" [date]="eventDate"></sw-countdown>
```

Comment:
I think that there's no need to add a scrollspy or id for this, since we are not referencing this in the navbar menu